src/main/java/seedu/ddd/model/event/Event.java

```diff
@@ -0,0 +1,112 @@
+package seedu.ddd.model.event;
+
+import static seedu.ddd.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+
+import seedu.ddd.commons.util.ToStringBuilder;
+import seedu.ddd.model.contact.client.Client;
+import seedu.ddd.model.contact.vendor.Vendor;
+
+/**
+ * Represents a Event in the address book.
+ * Guarantees: to be determined.
+ */
+public class Event {
+
+    private final ArrayList<Client> clients;
+    private final ArrayList<Vendor> vendors;
+    private final String eventDescription;
+
+    /**
+     * Constructs a {@code Event}.
+     *
+     * @param clients A list of client.
+     * @param vendors A list of vendors.
+     */
+    public Event(ArrayList<Client> clients, ArrayList<Vendor> vendors, String description) {
+        requireAllNonNull(clients, vendors);
+        // another test to check
+        this.clients = clients;
+        this.vendors = vendors;
+        this.eventDescription = description;
+    }
+
+    /**
+     * Returns the clients list.
+     * @return An {@code ArrayList} of {@code Client}.
+     */
+    public ArrayList<Client> getClients() {
+        return clients;
+    }
+
+    /**
+     * Returns the vendors list.
+     * @return An {@code ArrayList} of {@code Vendor}.
+     */
+    public ArrayList<Vendor> getVendors() {
+        return vendors;
+    }
+
+    /**
+     * Returns the event description.
+     * @return A {@code String} which represents the description of the event.
+     */
+    public String getEventDescription() {
+        return this.eventDescription;
+    }
+
+    /**
+     * Return if the two events are the same.
+     * @param otherEvent Another event.
+     * @return A boolean value which represents the result.
+     */
+    public boolean isSameEvent(Event otherEvent) {
+        return this.equals(otherEvent);
+    }
```

Comment:
I think we don't need this method. I know `Contact` has an `isSameContact` method, but I don't see the need for it.

Reply:
@leslieyip02 Why not to prevent creation of duplicate events?

Reply:
I think its redundant because this just calls `this.equals`.

Reply:
Oh. In that case @FYL2003 can you implement it such that it checks if the clients are the same? (What are the odds that the clients are planning 2 weddings concurrently lol)